src/core/polyhedral/mapped_scop.cc

```diff
@@ -572,6 +573,9 @@ std::tuple<std::string, tc::Grid, tc::Block> MappedScop::codegen(
   code << "extern \"C\" {" << std::endl
        << emitCudaKernel(specializedName, *mappedScopForCodegen) << "}"
        << std::endl;
+#else
+  code << "TC was compiled with -DWITH_CUDA=0 which disabled cuda compilation";
+#endif
```

Comment:
The next commit _reverts_ these changes.
If they are not needed, then don't apply them first.
Btw, I seem to be able to compile with `-DWITH_CUDA=0` without these changes.